news.html

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: News
+subtitle: Check out our latest work, notable events, and ongoing studies!
+---
+
+<div>
+  {% for post in site.posts %}
+  <article class="pb2 measure-wide lh-copy">
+    <h2 class="mb2"><a href="{{ post.url }}">{{ post.title }}</a></h2>
+    <h4 class="mt0">{{ post.date | date: "%B %-d, %Y" }}</h4>
+    <p>{{ post.content }}</p>
+    {%if post.studyLink %}
+    <a href="{{post.studyLink}}" class="black link hover-cmu-red mr3 dib">
+      <span class="cta">Participate</span>
+    </a>
+    {%endif%}
+  </article>
+  <hr />
```

Comment:
Let's instead add some space.

Reply:
added some but might be too much

Reply:
Did you look at other spacings on the website? I don't remember exactly the principle I used to determine spacing on the website but maybe you can find the pattern 😇.

Reply:
Wasn't really one, but made it smaller, #createthepattern